deltachat-ios/Controller/ChatListController.swift

```diff
@@ -101,9 +101,14 @@ class ChatListController: UITableViewController {
 
         // update messages - for new messages, do not reuse or modify strings but create new ones.
         // it is not needed to keep all past update messages, however, when deleted, also the strings should be deleted.
-        //let msg = dcContext.newMessage(viewType: DC_MSG_TEXT)
-        //msg.text = String.localized("update_1_30")
-        //dcContext.addDeviceMessage(label: "update_1_30a_ios", msg: msg)
+        let msg = dcContext.newMessage(viewType: DC_MSG_TEXT)
+        msg.text = "1.32 Highlights:\n\n"
+            +   "• Multi-select in chat list\n"
+            +   "• Reply privately\n"
+            +   "• Search in full message view\n"
+            +   "• Search in help\n"
+            +   "• Improve overall accessibility ✨"
```

Comment:
ups, I saw this too late, I would probably remove that entry. There are too many unaddressed accessibility issues that users have reported, it might be missleading to them.